### PR TITLE
[PKG-2207] kaleido-core 0.2.1

### PR DESCRIPTION
**Pull request by lorepirri**
[PKG-2207] kaleido-core 0.2.1

- fix linter issues
- pin `host` section

[PKG-2207]: https://anaconda.atlassian.net/browse/PKG-2207?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lorepirri** [comment]
@akabanovs you can take over, your dependency `mathjax =2.7` is missing, but the linter is fixed.

I am not sure if the `host` pinned versions are working as I am not able to build without the missing dependency.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,19 +37,17 @@ requirements:
...
   run:
     - expat   # [linux]
     - nspr    # [linux]
     - nss     # [linux]
     - sqlite  # [linux]
     - fontconfig         # [linux]
-    - fonts-conda-forge  # [linux]
-    - __osx >=10.11      # [osx]
-    - mathjax =2.7
+    - mathjax 2.7.5
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: In logs:
`Adding .* to spec 'mathjax 2.7.5' to ensure satisfiability.  Please consider putting {{ var_name }}.* or some relational operator (>/</>=/<=) on this spec in meta.yaml, or if req is also a build req, using {{ pin_compatible() }} jinja2 function instead.  See https://conda.io/docs/user-guide/tasks/build-packages/variants.html#pinning-at-the-variant-level`

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**